pkg/controller/controller.go

```diff
@@ -390,6 +392,19 @@ func (c *Controller) RunOnce() error {
 	for _, nodeGroupOpts := range c.Opts.NodeGroups {
 		log.Debugf("**********[START NODEGROUP %v]**********", nodeGroupOpts.Name)
 		state := c.nodeGroups[nodeGroupOpts.Name]
+		// Double check if node group still exists from the cloud provider then retrieve the latest stat
+		cloudProviderNodeGroup, ok := c.cloudProvider.GetNodeGroup(nodeGroupOpts.CloudProviderGroupName)
+		if !ok {
+  		err = errors.New("could not find node group!")
```

Comment:
Indenting should be 1 level in from if statement. Running 'go fmt' should fix this.

Reply:
sorry, my vim messed up. should have double check the code :)

Reply:
I've already signed the CLA.